src/Components/ArticlePage.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { fetchArticlesById } from "../api";
+
+const ArticlePage = () => {
+  const { article_id } = useParams();
+  const [isLoading, setIsLoading] = useState(true);
+  const [sArticle, setSArticle] = useState(null);
+
+  useEffect(() => {
+    setIsLoading(true);
+    fetchArticlesById(article_id).then((result) => {
+      setSArticle(result[0]);
+      setIsLoading(false);
+    });
+  }, [article_id]);
+
+  return isLoading ? (
+    <div key="Loading">
+      <p>Loading...</p>
+    </div>
+  ) : (
+    <div key={article_id} className="grid grid-cols-2 grid-rows-5 gap-1">
```

Comment:
having article_id as key on the parent div, as well as the component, might be the cause of the unique key error I'm seeing in the terminal.